some tests with engine=OpenDCS-XML don't work on windows

**Describe the bug**
some tests don't work properly in windows.

**To Reproduce**

```
gradlew  clean :testing:opendcs-tests:test -Pno.docs=true -Popendcs.test.engine=OpenDCS-XML
```

**Expected behavior**

should work on windows an ubuntu

**Screenshots**
```bash
 Task :testing:opendcs-tests:test

DbEnumDaoTestIT > test_write_enum() FAILED
    org.opendcs.database.api.OpenDcsDataException at DbEnumDaoTestIT.java:32

ScheduledEntryTest > test_scheduled_entry_delete() FAILED
    decodes.db.DatabaseException at DatabaseService.java:56
        Caused by: decodes.db.DatabaseException at XmlDatabaseProvider.java:53
            Caused by: decodes.db.DatabaseException at XmlDatabaseIO.java:700

ScheduledEntryTest > test_retrieve_entry_by_status_id() FAILED
    decodes.db.DatabaseException at DatabaseService.java:56
        Caused by: decodes.db.DatabaseException at XmlDatabaseProvider.java:53
            Caused by: decodes.db.DatabaseException at XmlDatabaseIO.java:700

DecodesTest > test_humanReadable(String, String) > [1] OKVI4-input, SimpleDecodesTest/golden${impl} FAILED
    decodes.db.DatabaseException at DatabaseService.java:56
        Caused by: decodes.db.DatabaseException at XmlDatabaseProvider.java:53
            Caused by: decodes.db.DatabaseException at XmlDatabaseIO.java:700

DecodesTest > test_HydroJsonTest(String, String) > [1] HydroJSON-Test, HydroJsonTest/golden${impl} FAILED
    decodes.db.DatabaseException at DatabaseService.java:56
        Caused by: decodes.db.DatabaseException at XmlDatabaseProvider.java:53
            Caused by: decodes.db.DatabaseException at XmlDatabaseIO.java:700

16 tests completed, 5 failed, 11 skipped
```

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -54,7 +54,6 @@ jobs:
         # Integration tests require docker now for test containers.
         # More work required to deal with the other platforms. Should be workable
         # at some point though.
-        # if: matrix.platform == 'ubuntu-latest'
         run: |
           ./gradlew :testing:opendcs-tests:test -P"opendcs.test.engine=OpenDCS-XML" --info
       - name: integration tests (OpenDCS-Postgres)

--- a/.github/workflows/build.yml
+++ b/.github/workflows/build.yml
@@ -54,7 +54,7 @@ jobs:
         # Integration tests require docker now for test containers.
         # More work required to deal with the other platforms. Should be workable
         # at some point though.
-        if: matrix.platform == 'ubuntu-latest'
+        # if: matrix.platform == 'ubuntu-latest'
         run: |
           ./gradlew :testing:opendcs-tests:test -P"opendcs.test.engine=OpenDCS-XML" --info
       - name: integration tests (OpenDCS-Postgres)

--- a/java/opendcs/src/main/java/org/opendcs/database/impl/xml/dao/EnumXmlDao.java
+++ b/java/opendcs/src/main/java/org/opendcs/database/impl/xml/dao/EnumXmlDao.java
@@ -265,7 +265,7 @@ public class EnumXmlDao implements EnumDAI
             File xmlFile = new File(conn.getDirectory(), "enum/EnumList.xml");
             try (OutputStream outStream = new FileOutputStream(xmlFile))
             {
-                XMLStreamWriter writer = f.createXMLStreamWriter(outStream);
+                XMLStreamWriter writer = f.createXMLStreamWriter(outStream,"UTF-8");
                 writer.writeStartDocument("UTF-8", "1.0");
                 writer.writeStartElement("EnumList");
                     for (DbEnum curDbEnum: existing)
@@ -313,6 +313,7 @@ public class EnumXmlDao implements EnumDAI
         }
         catch (IOException | XMLStreamException | SQLException ex)
         {
+            ex.printStackTrace();
             throw new OpenDcsDataException("Unable to write enum to XML database.",ex);
         }
     }

--- a/java/opendcs/src/main/java/org/opendcs/database/impl/xml/dao/EnumXmlDao.java
+++ b/java/opendcs/src/main/java/org/opendcs/database/impl/xml/dao/EnumXmlDao.java
@@ -313,7 +313,6 @@ public class EnumXmlDao implements EnumDAI
         }
         catch (IOException | XMLStreamException | SQLException ex)
         {
-            ex.printStackTrace();
             throw new OpenDcsDataException("Unable to write enum to XML database.",ex);
         }
     }

